ros_buildfarm/templates/snippet/publisher_publish-over-git.xml.em

```diff
@@ -0,0 +1,12 @@
+<hudson.plugins.git.GitPublisher plugin="git@@3.5.1">
+  <configVersion>2</configVersion>
+  <pushMerge>false</pushMerge>
+  <pushOnlyIfSuccess>true</pushOnlyIfSuccess>
+  <forcePush>false</forcePush>
+  <branchesToPush>
+    <hudson.plugins.git.GitPublisher_-BranchToPush>
+      <targetRepoName>origin</targetRepoName>
+      <branchName>@branch</branchName>
+    </hudson.plugins.git.GitPublisher_-BranchToPush>
+  </branchesToPush>
+</hudson.plugins.git.GitPublisher>
```

Comment:
The indentation is wrong resulting in a diff when regenerating the job. See #619 for the fix.